components/script/dom/history.rs

```diff
@@ -80,4 +149,22 @@ impl HistoryMethods for History {
     fn Forward(&self) {
         self.traverse_history(TraversalDirection::Forward(1));
     }
+
+    // https://html.spec.whatwg.org/multipage/#dom-history-pushtstae
```

Comment:
Woops, yeah a typo it should be https://html.spec.whatwg.org/multipage/#dom-history-pushstate